src/tlo/methods/rti.py

```diff
@@ -3961,14 +4048,11 @@ def apply(self, person_id, squeeze_factor):
         assert len(p['rt_injuries_to_cast']) > 0
         # Check this injury assigned to be treated here is actually had by the person
         assert all(injuries in person_injuries.values for injuries in p['rt_injuries_to_cast'])
-        # If they have a fracture that needs a cast, ask for plaster of paris
-        self.module.item_codes_for_consumables_required['fracture_treatment'] = {
-            get_item_code('Plaster of Paris (POP) 10cm x 7.5cm slab_12_CMST'): fracturecastcounts,
-            get_item_code('Bandage, crepe 7.5cm x 1.4m long , when stretched'): 200,
-        }
-        is_cons_available = self.get_consumables(
-            self.module.item_codes_for_consumables_required['fracture_treatment']
-        )
+
+        # If they have a fracture that needs a cast, ask for plaster of paris, update POP count
+        self.module.cons_item_codes['fracture_treatment'].update({
```

Comment:
This updated the module's store of the consumables, right? So would have consequences for all subsequent individuals. I think the interned behaviour originally was that these are additional
Items needed only for this person.